
This message shows early, breaking the nice bootsplash,
so we "hide" it as a debug message

https://bugs.mageia.org/show_bug.cgi?id=11532

Signed-off-by: Thomas Baklund <tmb@mageia.org>

--- linux/arch/x86/kernel/tsc.c.orig	2013-11-04 01:41:26.000000000 +0200
+++ linux/arch/x86/kernel/tsc.c	2013-11-08 22:18:39.366600972 +0200
@@ -386,7 +386,7 @@ static unsigned long quick_pit_calibrate
 			goto success;
 		}
 	}
-	pr_err("Fast TSC calibration failed\n");
+	pr_debug("Fast TSC calibration failed\n");
 	return 0;
 
 success:
